xmrstak/backend/nvidia/nvcc_code/cuda_extra.cu

```diff
@@ -418,6 +418,17 @@ extern "C" int cuda_get_deviceinfo(nvid_ctx* ctx)
 		
 		// no limit by default 1TiB
 		size_t maxMemUsage = byteToMiB * byteToMiB;
+	    if(props.major == 6)
```

Comment:
Can you fix the indentation here.

Also please don't use un-bracketed `if` on multi-line code - this makes it very hard to read.